Products/CMFEditions/StandardModifiers.py

```diff
@@ -28,6 +28,10 @@
 import os,sys
 from itertools import izip
 from App.class_init import InitializeClass
+from plone.dexterity.interfaces import IDexterityContent
```

Comment:
It doesn't seem so. How could I test between AT/Dexterity content types otherwise?